UI/frontend-plugins/frontend-tools/scripts.cpp

```diff
@@ -455,11 +470,28 @@ void ScriptsTool::on_pythonPathBrowse_clicked()
 
 	ui->pythonPath->setText(newPath);
 
-	if (obs_scripting_python_loaded())
+	bool loaded = obs_scripting_python_loaded();
+	bool pathChanged = !newPath.isEmpty() && curPath.compare(newPath) != 0;
```

Comment:
Minor nitpick, if this were moved into the `if` statement, then the comparisons and checks would only be evaluated when `loaded` is true. As-is, they are evaluated even if python is not loaded at all (when `loaded` is false).

Reply:
True. Fixed.